p2p/src/peer_manager/peerdb/mod.rs

```diff
@@ -259,7 +293,14 @@ impl<T: NetworkingService> PeerDb<T> {
                 // This can fail only if `SystemTime::now()` returns the time before `UNIX_EPOCH`.
```

Comment:
OK I can see there's an infection of SystemTime all over the place...

Reply:
I will change that.